### DISCUSSION TITLE
如果遇到買東西被多算錢要怎麼處理

### DISCUSSION THREAD

**Discussion by Jamesmoneya** [1101-GS4719-C/Python-Community-GS4719-C#1431] [General]
一直有個問題
超商現在很少有貼產品標價，如果沒有去問店員，不會知道產品價格

因為現在都用載具發票，發票明細不會馬上在app上出來，今天回家就發現飲料被多結帳
一罐（明明買一罐竟然刷成兩罐？？）

雖然才40元，不知道要不要特別回去要？

以前也有一次被多刷，當場發現不對反應，但趕時間的話不能每次都算好，這種回家才發
現被結錯的怎麼辦？？自己吞下去嗎

**Comment by TW-Maverick** [comment]
如果要避免上述事情的發生，我覺得可以事先算好你買的物品總共多少，這樣數字有誤差的話，也有個底。
當店員在刷物品時，可以盯著螢幕看金額對不對，我現在很少遇到被多算錢的情況，除了我估錯價錢😂
趕時間的話，其實也沒什麼辦法！

**Comment by alicia0725** [comment]
為了避免這種情況我通常會在結帳前算好是多少錢，如果不一樣也能當場發現，至於回家才發現的話，如果數額不大可能就會想說算了，但如果是大數額感覺還是要拿著電子發票去詢問看看

**Comment by Alexgns** [comment]
我的作法是店員刷條碼時專注在顯示的電腦屏幕上，如果出現和自己買的東西不相符時便能夠及時反應，避免爭議以及額外的損失

**Comment by Tzuhsuannnnn** [comment]
我認為可以在結帳前先預估一下自己的金額，如果和店員算的金額有出入時，可以看顯示螢幕確定一下有沒有算錯，當場發現算錯的話可以直接和店員反應。若是回家才發現，再拿回去反應可能會有點麻煩。

**Comment by Superlan8** [comment]
我會避免這種狀況發生，所以每次我結帳後，我都會站在櫃台，把錢算完才離開，我認為這是自己的權益，即便可能會被其他顧客注意，我也堅持這種作法，保護自己也被免被誤會。

**Comment by pigeon9975** [comment]
我會在結帳前先大概算一下價錢，然後結帳時盯著螢幕，確定該有的折扣都有算到，或是確認他沒有刷錯我的東西，如果這樣還是被算錯錢，那就只能認賠了QQ

**Comment by ALAN30359704** [comment]
我覺得最重要就是在結帳的時候就應該要確認好的商品，然後再買的時候你可以騎計算一下價格，這樣你就知道說螢幕顯示的金額跟你想的一不一樣，如果少的話可能就有特價，那如果多的話應該就是他有多刷或者是你需要再確認一下！

**Comment by alstinlin** [comment]
通常我會在結帳時先算好金額，因為之前朋友遇到的狀況是發票上也打錯品項，導致後續跟店家反映的時候被告知需要調監視器來確認，所以如果是當下就發現的話，就能夠避免時間或金錢的損失了~

**Comment by RogelioKG** [comment]
有沒有人有過買的飲料放在超商桌上，結果忘記拿走的經驗，如果是超商店員，那又應該如何處理？之前有一次有過這樣的經驗，注意到的時候已經超過兩個小時。原本打算拿著發票去要回，可是進去的時候發現店員已經輪班，忘在桌上的飲料似乎也被收回了，最後只好重新再買一杯。不知道遇到這樣子的狀況要如何解決，求解QQ

**Comment by 111607004** [comment]
我都會看店員刷條碼之後會顯示的那個螢幕，也會算一下大概多少錢，這樣就可以避免爭議。但我每次都買一樣的東西所以很熟悉價格

**Comment by poyen0806** [comment]
我都在結帳前就先算好要付多少錢
如果沒有特別標價的話我就會死盯著螢幕

但其實可以用request和bs4去爬取一般店家的價格去做參考
這樣就比較不會被坑了

**Comment by irenesunnn** [comment]
先從結論說：我建議還是要回去處理。雖然 40 元看起來不多，但從長期來看這其實是個消費者權益的問題。如果每個人都覺得"小錢就算了"，可能反而會讓這種問題一直存在。
而且就我的經驗，超商店員通常都很願意處理這類問題，畢竟多收確實是他們的疏失。帶著載具跟發票去，他們可以很快查到紀錄。
不過也認同樓主說的，不可能每次結帳都仔細核對。建議可以養成看一下總金額的習慣，感覺不對就問一下。我現在都會順手截個螢幕，這樣回家比對比較方便。
最後想說，不用覺得為了小錢去麻煩店員不好意思。這不只是錢的問題，更是讓經營者注意到問題所在，才能避免類似情況一直發生。

**Comment by XiangLuen** [comment]
如果是我的話，40 塊其實不算小錢，我會回去要，尤其是如果這家超商是你常去的地方。因為這種事發生一次可能是店員疏忽，但如果發生兩次、三次，那就真的有問題了。而且，這種錯誤其實店員自己也不一定會發現，長期下來可能很多客人都被多收卻沒去反應，最後只是讓店家白賺。

**Comment by SendaiHatsuki** [comment]
當然是硬氣起來，讓店員重新算錢，別人的錯誤不該由我們來買單，但如果今天是出國發生這種事，以我的經驗我應該會就算了，畢竟語言不通真的不如少一事會比較輕鬆。

**Comment by lichilun** [comment]
如果發現自己被多算錢。我覺得，首先先檢查一下發票或者帳單，確定到底是什麼地方錯了，然後，冷靜地跟店員說明情況，告訴他們多收的錢是哪部分，通常店員會幫忙處理，但假如不幫忙處理，（像我之前吃火鍋被多收100元），可能需要走法律程序了.......

**Comment by kai1660** [comment]
用日語跟司機講清楚

**Comment by ryan8967** [comment]
我個人通常在結帳前會粗估算一下價錢，若相差不大就不會回去要了，畢竟時間也是成本。但如果同家店發生不只一次則代表這家有問題，當下我會向店家反映

**Comment by BOSENCHANG2004** [comment]
這問題的確是非常常見的問題，而當下遇到可能真的非常容易不知所措。
但是這種時候一定要鼓起勇氣去問店員，店家那邊一定會有紀錄。
這是我們消費者的權益絕對不要害怕!!

**Comment by Kaz9487** [comment]
這個問題非常常見，但很容易不知道該怎麼做，特別是有時會怕麻煩就直接將錯就錯。
如果是小額又是我多付錢的話，我會覺得那就算了，來回的時間都不符時間成本。
如果是大額的話，這是我們消費者的權利，應該去跟店家反應並拿回退款或補償。
